src/rollback_to_stable/rts_btree_walk.c

```diff
@@ -458,5 +458,11 @@ __wti_rts_btree_walk_btree(WT_SESSION_IMPL *session, wt_timestamp_t rollback_tim
     if (btree->root.page == NULL)
         return (0);
 
-    return (__rts_btree_walk(session, rollback_timestamp));
+    WT_RET(__rts_btree_walk(session, rollback_timestamp));
+
+    // TODO - comment
```

Comment:
Are we ok with resetting both to 0? Both are used in [this](https://github.com/wiredtiger/wiredtiger/blob/c0a2ab1586e74d7c727772305350ce995517e238/src/btree/bt_sync.c#L377-L381) check.

Reply:
Yes.

Reply:
@bvpvamsikrishna suggests resetting to the stable timestamp and max txn id of the stable snapshot, see [here](https://jira.mongodb.org/browse/BF-35100?focusedCommentId=6720034&focusedId=6720034&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-6720034). What is the correct approach? If we reset to 0, we will always satisfy the visibility checks. What if we set those values to the same of the stable snapshot?

Reply:
We are unsure what the maximum transaction id or timestamp exists on that table. We only know that RTS can ensure that it has less than the maximum transaction id or timestamp.  I feel we shouldn't set them to the stable timestamp.

Reply:
I will still wait for @bvpvamsikrishna's opinion.
In the meantime, I updated the associated comment, see b250afff0f4413f4f75b5c43579d3306a39cc2ec.

Reply:
My preference is not set them to '0', instead set them to stable / checkpoint timestamp and the txn id as (snap_min -1) of txn snapshot.
'0' represents global visibility, and I think it might be dangerous for the cases, when the standby is promoted to primary scenario's.
CC : @kommiharibabu , @etienneptl